packages/logger/CHANGELOG.md

```diff
@@ -0,0 +1,108 @@
+# Changelog
```

Comment:
this changelog is probably not so relevant. We might rename it to keep the history, but it makes sense to start a new one for this particular module